docs/dev/intellij/intelliJ.md

```diff
@@ -0,0 +1,53 @@
+# IntelliJ
+
+## Checkstyle
+
+This IJ add checkstyle warnings an errors.
```

Comment:
grammar is all over the place: `There is an IntelliJ Plugin to see Checkstyle errors as errors in your IDE`